browser/ui/webui/brave_new_tab_ui.cc

```diff
@@ -62,10 +62,12 @@ void BraveNewTabUI::SetStatsWebUIProperties(
     render_view_host->SetWebUIProperty(
         "fingerprintingBlockedStat",
         std::to_string(prefs->GetUint64(kFingerprintingBlocked)));
+#if !defined(OS_ANDROID)
```

Comment:
this seems weird, does android use brave_new_tab_ui?

Reply:
yeah, it pulls it, I tried to disable in gn files, but there are other errors in that case. Need to go deeper to get an answer.